src/Panel.vue

```diff
@@ -340,10 +340,12 @@
             this.insertCaretInsideHeader(this.$refs.headerWrapper.innerHTML);
         }
       });
-      const panelHeader = this.$slots.header ? this.$refs.headerWrapper.innerHTML : this.headerContent;
-      const panelHeaderText = jQuery(panelHeader).wrap('<div></div>').parent().find(':header').text();
-      if (panelHeaderText) {
+
+      if (this.headerContent) {
+        const panelHeaderText = jQuery(this.headerContent).wrap('<div></div>').parent().find(':header').text();
         this.$refs.cardContainer.setAttribute('id', string(panelHeaderText).slugify().toString());
+      } else if(this.$refs.headerWrapper.innerHTML) {
+        this.$refs.cardContainer.setAttribute('id', jQuery(this.$refs.headerWrapper.innerHTML).wrap('<div></div>').parent().find(':header')[0].id);
```

Comment:
Will this cause an error if there is no header at all?

Maybe try something like this instead?

```javascript
const header = this.$refs.headerWrapper.innerHTML).wrap('<div></div>').parent().find(':header');
if (header.length > 0) {
  this.$refs.cardContainer.setAttribute('id', header[0].id);
}
```